src/main.ts

```diff
@@ -9,11 +9,17 @@ import { AppModule } from './app.module';
 import { AllExceptionsFilterLogger } from './shared/exception-filters/http-exceptions-logger.filter';
 import { winstonLoggerOptions } from './logging/winston.options';
 import { LoggingInterceptor } from './logging/logging.interceptor';
+import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  // add global prefix
   app.setGlobalPrefix('api');
 
+  // add global validation pipe
+  app.useGlobalPipes(new ValidationPipe());
+
   // Enable cors
   app.enableCors();
 
```

Comment:
I've seen in the docs of Nestjs that we must first create and initialize a microservice using  `await NestFactory.createMicroservice`
Has this part been replaced and made using another method/technique? Because I can't find it.

Reply:
There are multiple ways to emit message and make the whole application microservice or just the module that we will send from.